src/api/Comments.js

```diff
@@ -40,6 +50,143 @@ class Comments extends OffsetBasedAPI {
 
         this.successCallback({ ...data, entries: comments });
     };
+
+    /**
+     * API URL for comments endpoint
+     *
+     * @param {string} [id] - A box comment file id
+     * @return {string} base url for comments
+     */
+    getCommentsUrl(id?: string): string {
+        const suffix: string = id ? `/${id}` : '';
```

Comment:
Maybe can make it look prettier

const url = ${this.getBaseApiUrl()}/comments
return id ? url/{id} : url